### PR TITLE
docs(rpc): add info about encoding

### PR DESCRIPTION
**Pull request by melekes**
---

#### PR checklist

- [ ] ~~Tests written/updated~~
- [ ] ~~Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)~~
- [x] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [x] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

File: rpc/openapi/openapi.yaml [hunk 1]
```diff
@@ -54,6 +54,17 @@ info:
     However, the support for unsuffixed methods will be dropped in future releases and all users are
     encouraged to migrate as soon as possible.
 
+    ## Encoding
```
Comment (cason) [inline rpc/openapi/openapi.yaml:57]: Is it not base64?

Comment (melekes) [reply rpc/openapi/openapi.yaml:57]: What are you referring to? If you're speaking about byte arrays, then yes.

Comment (cason) [reply rpc/openapi/openapi.yaml:57]: I know we have to encode txs in base64 format, this is the amino JSON format?

**Review by cason — APPROVED** [review]
+1

**Pull request merged**